apisix/plugins/ext-plugin/init.lua

```diff
@@ -611,6 +611,29 @@ local rpc_handlers = {
                 end
             end
 
+            local len = rewrite:RespHeadersLength()
+            local exclude_resp_header = {
```

Comment:
Let's move the table in the module level

Reply:
I understand to put it in the 'apisix/constants. lua' file, right?

Reply:
You can move to the top of the current file.